pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
```

Comment:
nit: blank line between text and code block. I've the lack of spacing confuse some markdown renderers.

Same throughout and same for headers+paragraphs, etc.